post-types/templates/single-tabular.php

```diff
@@ -250,10 +265,10 @@
 		"bAutoWidth": false,
     dom: '<"top"<"info"i><"pagination"p><"length"l>>rt',
     lengthMenu: [[10, 25, 50, -1], [10, 25, 50, "All"]],
-    order: [[ <?php echo isset($group_data_by_column_index) && $group_data_by_column_index != '' ?  $group_data_by_column_index : 0 ?>, 'asc' ]],
+    order: [[ 0, 'asc' ]],
```

Comment:
@Huyeng your PR removes some code changes that I did which are very important:
- allowing to group and sort rows by a certain column